hail/build.gradle

```diff
@@ -44,7 +44,7 @@ String hailShortVersion = MAJOR.toString() + "." + MINOR.toString()
 String hailPipVersion = hailShortVersion + "." + PATCH + System.getProperty("hail.pip-version-suffix", "")
 String[] deployedSparkVersions = new File("deployed-spark-versions.txt").readLines()
 
-String sparkVersion = System.getProperty("spark.version", "2.2.0")
+String sparkVersion = System.getProperty("spark.version", "2.4.0")
```

Comment:
the fail message below should be 2.4

Reply:
Changing the default is a backwards incompatible change to our build system. Not sure how we feel about doing that to those that build on us.